src/components/screens/ScoreSetCreator.vue

```diff
@@ -0,0 +1,2021 @@
+    ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+    resetForm: function () {
+      if (this.item) {
+        this.experiment = this.item.experiment
+        this.licenseId = this.item.license.id
+        // metaAnalyzesScoreSets is only used when editing a new score set, so we don't need to populate it from URNs.
+        this.metaAnalyzesScoreSets = []
+        this.supersededScoreSet = this.item.supersededScoreSet
+        this.title = this.item.title
+        this.shortDescription = this.item.shortDescription
+        this.abstractText = this.item.abstractText
+        this.methodText = this.item.methodText
+        this.keywords = this.item.keywords
+        this.doiIdentifiers = this.item.doiIdentifiers
+        // So that the multiselect can populate correctly, build the primary publication identifiers
+        // indirectly by filtering a merged list of secondary and primary publication identifiers
+        this.publicationIdentifiers = _.concat(this.item.primaryPublicationIdentifiers, this.item.secondaryPublicationIdentifiers)
+        this.primaryPublicationIdentifiers = this.item.primaryPublicationIdentifiers.filter((publication) => {
+          return this.publicationIdentifiers.some((primary) => {
+            return primary.identifier === publication.identifier
+          })
+        })
+        this.secondaryPublicationIdentifiers = this.item.secondaryPublicationIdentifiers
+        this.dataUsagePolicy = this.item.dataUsagePolicy
+        this.taxonomy = this.item.taxonomy
+        this.targetGene = emptyTargetGene()
+        this.assembly = this.item.assembly
+        this.targetGenes = this.item.targetGenes
+        this.extraMetadata = this.item.extraMetadata
+      } else {
+        this.experiment = null
+        this.licenseId = this.defaultLicenseId
+        this.metaAnalyzesScoreSets = []
+        this.supersededScoreSet = null
+        this.title = null
+        this.shortDescription = null
+        this.abstractText = null
+        this.methodText = null
+        this.keywords = []
+        this.doiIdentifiers = []
+        this.primaryPublicationIdentifiers = []
+        this.secondaryPublicationIdentifiers = []
+        this.publicationIdentifiers = []
+        this.dataUsagePolicy = null
+        this.taxonomy = null
+        this.extraMetadata = {}
+        this.resetTarget()
+        this.targetGenes = []
+      }
+    },
+
+    resetTarget: function () {
+      this.taxonomy = null
+      this.assembly = null
+      this.assemblySuggestions = []
+      this.assemblyDropdownValue = null
+      this.existingTargetGene = null
+      this.geneName = null
+      this.geneNameAccessionSuggestions = []
+      this.geneNameDropdownValue =  null
+      this.fileCleared('targetGeneTargetSequenceSequenceFile')
+      this.referenceGenome = null
+      this.targetGene = emptyTargetGene()
+    },
+
+    addTarget: function () {
```

Comment:
We'll need to integrate https://github.com/VariantEffect/mavedb-ui/pull/194 here. I can handle that in the other PR once this one is merged in.